apm-agent-core/src/main/java/co/elastic/apm/agent/bci/IndyBootstrap.java

```diff
@@ -441,6 +451,7 @@ private static ConstantCallSite internalBootstrap(MethodHandles.Lookup lookup, S
             MethodHandles.Lookup indyLookup = (MethodHandles.Lookup) lookupExposer.getMethod("getLookup").invoke(null);
             // When calling findStatic now, the lookup class will be one that is loaded by the plugin class loader
             MethodHandle methodHandle = indyLookup.findStatic(adviceInPluginCL, adviceMethodName, adviceMethodType);
+            methodHandle = wrapWithExceptionCatcher(methodHandle);
```

Comment:
[For Reviewer]
In the related user issue I noticed that uncaught exceptions from advices are ONLY logged to stdout.
I don't know if this bit us already at the past, but I feared it might in the future.
When the exception is only logged to stdout it (a) can confuse users and (b) means that the agent logs don't show us the full story. We would most likely see messages about orphaned spans with no idea where they come from without also having the stdout of the application.

I therefore thought it might be a good idea to improve this behaviour. With this change:
 * Exceptions are caught (they were caught before aswell) and logged to the agent logs
 * Exceptions are not logged to stdout anymore
 
 Essentially this change is equivalent to adding a `try {} catch(Throwable t){ log.error(t);}` around all advices. The performance might be slightly worse on Java 7 (additional method call), on Java 8+ it should be the same as directly inling the try-catch ([source](https://www.youtube.com/watch?v=rhBfvWQRe0w) making me think this).
 
 If you feel like this change is to risky performance wise, don't hesitate to speak up. I'm fine with removing this from this PR.